lib/generators/templates/ci/ci.yml.tt

```diff
@@ -86,6 +86,7 @@ jobs:
     runs-on: ubuntu-latest
 
     services:
+      <%- if using_postgres? -%>
```

Comment:
Should this block be tested that it is present for Postgres and absent for SQLite?